cfme/tests/configure/test_default_views_infra.py

```diff
@@ -27,6 +29,17 @@
 }
 
 
+def test_default_view_infra_reset(appliance, soft_assert):
+    """
+        This test case performs Reset button test.
+    """
+    navigate_to(appliance.user.my_settings, "DefaultViews")
+    view = appliance.browser.create_view(DefaultViewsForm)
+    soft_assert(view.reset.disabled is True)
+    view.infrastructure.infrastructure_providers.select_button('Grid View')
```

Comment:
How would know default selected view is other than `Grid View`? I think you should first check for a currently selected view and then go for random selection between remaining two views. Same for cloud as well.